crates/red_knot_python_semantic/src/semantic_index/use_def/symbol_state.rs

```diff
@@ -51,9 +53,13 @@ use smallvec::SmallVec;
 #[newtype_index]
 pub(super) struct ScopedDefinitionId;
 
+impl ScopedDefinitionId {
+    pub(super) const UNBOUND: ScopedDefinitionId = ScopedDefinitionId::from_u32(0);
```

Comment:
Could you add some documentation for what `UNBOUND` is used and why it's okay to use 0?